exercises/acronym/example.jl

```diff
@@ -0,0 +1,5 @@
+#example.jl
+function myacronym(phrase)
+    regex = "[A-Z]+['a-z]*|['a-z]+"
+    return join([uppercase(word.match[1]) for word in eachmatch(Regex(regex), phrase)], "")
+end
```

Comment:
Please add a trailing newline to the end of the file. Makes it easier to read additions later on because it won't show the line containing end as changed